File: Base/Base_Tools45/Base_Tools45/Startup.cs
using Autodesk.AutoCAD.Runtime;

[assembly: ExtensionApplication(typeof(Base_Tools45.Startup))]

namespace Base_Tools45
{
    public class Startup : IExtensionApplication
    {
        public void Initialize()
        {
            BaseObjs._editor.WriteMessage("\nBase_Tools45.dll Loaded\n");
        }

        public void Terminate()
        {
        }
    }
}

File: Base/Base_VB/Base_VB/Startup.vb
Imports Autodesk.AutoCAD.Runtime
Imports Autodesk.AutoCAD.ApplicationServices

<Assembly: ExtensionApplication(GetType(Base_VB.Startup))> 

Public Class Startup
    Implements IExtensionApplication


    Public Sub Initialize() Implements IExtensionApplication.Initialize
        Application.DocumentManager.MdiActiveDocument.Editor.WriteMessage(vbLf & "Base_VB.dll Loaded" & vbLf)
    End Sub

    Public Sub Terminate() Implements IExtensionApplication.Terminate

    End Sub
End Class
